Server/middleware/verifyJWT.js

```diff
@@ -1,7 +1,10 @@
 import jwt from "jsonwebtoken";
+import { errorMessages } from "../utils/messages.js";
+import { handleError } from "../controllers/controllerUtils.js";
```

Comment:
We don't really need `handleError` here, all it does is add status and service and a default status code to the Error if they don't already exist.

We've already added that information, so we can just call `next(error)`

Reply:
Sure. let me just update it.